src/TransCreator/AmountInput.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { MaterialIcons } from '@expo/vector-icons';
+import { Input, Text } from 'native-base';
+import { Platform } from 'react-native';
+import styles from './Style';
+import numValidator from './numericInputValidator';
+
+const AmountInput = ({ transAmount, setTransAmount }) => (
+  <>
+    <MaterialIcons name="attach-money" style={styles.amountInputIcon} />
+    <Text>General</Text>
+    <Input
```

Comment:
can we empty the input when it gets focus and its value is 0? - it is not important, but a bit strange now